gpt4all-chat/chat.h

```diff
@@ -60,6 +60,8 @@ class Chat : public QObject
     Q_INVOKABLE void stopGenerating();
     Q_INVOKABLE void newPromptResponsePair(const QString &prompt);
 
+    QList<ResultInfo> results() const;
```

Comment:
I know this was already here and is just being made public , but since it is public I think it'd be nicer for the name to be a bit more descriptive than `results`, if I'm not familiar with the localdocs stuff I wouldn't necessary guess this was related to localdocs